classes/hazelcast-local-config.template.xml

```diff
@@ -2,38 +2,33 @@
 <hazelcast xmlns="http://www.hazelcast.com/schema/config"
```

Comment:
We'll need clear instructions for upgraders from older versions, as it looks like the clustering config has changed, and and although upgraders will get a new template, they'll need to update their actual config manually.